userdetails-plugin/grails-app/views/registration/createAccount.gsp

```diff
@@ -159,6 +159,31 @@
                         </div>
                     </div>
                 </g:if>
+                <g:if test="${edit}">
+                    <h2><g:message code="update.email" /></h2>
+                    <p>
+                        <g:if test="${grailsApplication.config.getProperty('userdetails.cognito.auth', boolean, false)}">
+                            <g:message code="update.email.desc.1" />
+                        </g:if>
+                        <g:else>
+                            <g:message code="update.email.desc.2" />
+                        </g:else>
+                    </p>
+                    <div id="newEmailDiv">
+                        <p id="newEmailMessage" hidden></p>
+                        <input id="newEmail" name="newEmail" type="text" class="form-control" data-validation-engine="validate[required]"/>
+                        <g:if test="${grailsApplication.config.getProperty('userdetails.cognito.auth', boolean, false)}">
```

Comment:
I would prefer this to be a feature based check rather than a cognito check - ie `userdetails.activation.enabled`, `userdetails.activation.manual: true` or similar, then remove `userdetails.cognito.auth` from properties (applicationForm will need to change too, use `userdetails.applications.insecure-urls-allowed` or similar)